tests/test_payments_client.py

```diff
@@ -14,7 +15,7 @@
 class PaymentsClientTests(CheckoutSdkTestCase):
     def setUp(self):
         super().setUp()
-        self.http_client = HttpClient(Config())
+        self.http_client = HttpClient(Config(secret_key="sk_test_4576e25e-1e16-49a7-8f85-e91c3291fc09"))
```

Comment:
secret_key has to be removed here. It is used to test things locally.

Reply:
Absolutely correct, thanks for catching this @dreamPathsProjekt!